<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpekerTestWdg</class>
 <widget class="QWidget" name="SpekerTestWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpekerTestWdg</string>
  </property>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>你正在使用的声音输出设备是：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSpeakerList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请点击&quot;播放测试音&quot;按钮试听：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartTest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>播放测试音</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>如果你未听到声音，可能存在以下原因：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1、扬声器或耳机未插好。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2、选择了错误的扬声器插孔，扬声器插孔通常为绿色。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3、扬声器未打开，或音量调节过小</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="navLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>442</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
